README.md

```diff
@@ -11,6 +11,7 @@ If you are not already using Inherited Resources we suggest instead using Rails'
 
 [![Version](http://img.shields.io/gem/v/inherited_resources.svg)](https://rubygems.org/gems/inherited_resources)
 [![Travis CI](http://img.shields.io/travis/activeadmin/inherited_resources/master.svg)](https://travis-ci.org/activeadmin/inherited_resources)
+[![Tidelift        ][tidelift_badge]][tidelift]
```

Comment:
I would either use the same style as existing badges, or extract all bagde links to the bottom.